easyfix-dictionary/src/lib.rs

```diff
@@ -170,6 +171,7 @@ impl TryFrom<&str> for BasicType {
             "EXCHANGE" => Ok(BasicType::Exchange),
             "FLOAT" => Ok(BasicType::Float),
             "INT" => Ok(BasicType::Int),
+            "LONG" => Ok(BasicType::Long),
```

Comment:
If `"LONG"` is only alias for `"INT"`, you can map it to `BasicType::Int` here and remove other changes.